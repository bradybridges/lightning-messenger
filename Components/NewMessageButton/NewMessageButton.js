import React from 'react';
import { Text, TouchableOpacity, Image, StyleSheet } from 'react-native';
import * as Constants from '../../Constants/Constants';
const add = require('../../assets/add.png');

const NewMessageButton = ({ toggleNewConversation }) => {
  return (
    <TouchableOpacity style={styles.container} onPress={() => toggleNewConversation()}>
       <Image source={add} style={styles.image} />
    </TouchableOpacity>
  );
}

const styles = StyleSheet.create({
  container: {
    width: 50,
    height: 50,
    position: 'absolute',
    bottom: 15,
    right: 15,
    backgroundColor: Constants.primaryHeaderColor,
    display: 'flex',
    alignItems: 'center',
    justifyContent: 'center',
    borderRadius: 100,
    borderColor: Constants.tertiaryBgColor,
    borderWidth: 1,
  },
  image: {
    width: 30, 
    height: 30,
  },
});


export default NewMessageButton
