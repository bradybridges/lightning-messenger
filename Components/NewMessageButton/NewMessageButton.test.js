import React from 'react';
import { shallow } from 'enzyme';
import NewMessageButton from './NewMessageButton';

describe('NewMessageButton', () => {
  let wrapper;
  it('should match the snapshot', () => {
    wrapper = shallow(<NewMessageButton />);
    expect(wrapper).toMatchSnapshot();
  });
});
