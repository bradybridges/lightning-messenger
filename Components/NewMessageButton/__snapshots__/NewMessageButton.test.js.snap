// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NewMessageButton should match the snapshot 1`] = `
<TouchableOpacity
  activeOpacity={0.2}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#121716",
      "borderColor": "#6E8F76",
      "borderRadius": 100,
      "borderWidth": 1,
      "bottom": 15,
      "display": "flex",
      "height": 60,
      "justifyContent": "center",
      "position": "absolute",
      "right": 15,
      "width": 60,
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 35,
        "width": 35,
      }
    }
  />
</TouchableOpacity>
`;
